modules/observe/scorecards/proc-installing-scorecard-plugin-in-rhdh-instance.adoc

```diff
@@ -0,0 +1,40 @@
+:_mod-docs-content-type: PROCEDURE
+
+[id="proc-installing-scorecard-plugin-in-rhdh-instance_{context}"]
+= Installing the Scorecard plugin in your {product} instance
+
+You must manually install and enable the plugin in your {product} instance.
+
+.Prerequisites
+* You have installed your {product-very-short} instance.
+* You have provisioned your custom {installing-and-viewing-plugins-book-link}[dynamic plugins config map].
+
+.Procedure
+
+* Add the following configuration in your `{product} dynamic-plugin-config.yaml` file:
```

Comment:
use backticks only for the file name. What's the correct file name: `{product} dynamic-plugin-config.yaml` file or `dynamic-plugin-config.yaml` file.
Global suggestion, look at other places as well in the doc.